p2p/src/net/mock/backend.rs

```diff
@@ -623,13 +641,13 @@ where
             .boxed(),
             Command::SendRequest {
                 peer_id,
+                request_id,
                 message,
-                response,
             } => async move {
                 boxed_cb(move |this| {
                     async move {
-                        let res = this.send_request(peer_id, message).await;
-                        response.send(res).map_err(|_| P2pError::ChannelClosed)
+                        let _res = this.send_request(request_id, peer_id, message).await;
```

Comment:
Would it make sense to log potential errors here and below?

Reply:
Sure, logs added.